package.json

```diff
@@ -5,15 +5,15 @@
   "main": "index.js",
   "scripts": {
     "lint": "eslint .",
-    "watcher:signature-request": "node src/watcher.js signature-request-watcher.config.js",
-    "watcher:collected-signatures": "node src/watcher.js collected-signatures-watcher.config.js",
-    "watcher:affirmation-request": "node src/watcher.js affirmation-request-watcher.config.js",
-    "sender:home": "node src/sender.js home-sender.config.js",
-    "sender:foreign": "node src/sender.js foreign-sender.config.js",
+    "watcher:signature-request": "node src/watcher.js signature-request-watcher.config.js | tee -a logs/watcher-signature-request.txt | scripts/prettify.sh",
```

Comment:
If this is a generic command line, is it worth to introduce a script `start-bridge-worker.sh` with the following content?

```bash
#!/usr/bin/env bash

WORKERS_DIR="src/"
LOGS_DIR="logs/"

WORKER="${WORKERS_DIR}${1}"
CONFIG="${2}.config.js"
LOG="${LOGS_DIR}${2}.txt"

if [ "${NODE_ENV}" = "production" ]; then
  exec node ${WORKER} ${CONFIG}
else
  node ${WORKER} ${CONFIG} | tee -a ${LOG} | pino-pretty
fi
```

This script will manage processes on a production system more thriftily. 

So, `package.json` should be modified to contain:

```json
"watcher:signature-request": "scripts/start-bridge-worker.sh watcher.js signature-request-watcher",
"watcher:collected-signatures": "scripts/start-bridge-worker.sh watcher.js collected-signatures-watcher",
"watcher:affirmation-request": "scripts/start-bridge-worker.sh watcher.js affirmation-request-watcher",
"sender:home": "scripts/start-bridge-worker.sh sender.js home-sender.config.js",
"sender:foreign": "node scripts/start-bridge-worker.sh sender.js foreign-sender", 
```

The corresponding changes needs to be done in `scripts/compute-stats.js` to work with correct log file names.

Reply:
Totally agree with this. I added it (with some minor changes) and updated the `compute-stats` script.

My only concern is that the scripts will need to be executed always from the root of the project (since bash scripts use filenames relative to the cwd of the process). It's not a big issue, just something to keep in mind. (There are some [tricks](https://stackoverflow.com/questions/6659689/referring-to-a-file-relative-to-executing-script) that can be used, or the script can be ported to [shelljs](https://github.com/shelljs/shelljs), but I think this is enough for now).